mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -247,7 +247,8 @@ class MultilayerPerceptronClassifier @Since("1.5.0") (
     }
     trainer.setStackSize($(blockSize))
     val mlpModel = trainer.train(data)
-    new MultilayerPerceptronClassificationModel(uid, myLayers, mlpModel.weights)
+    val model = new MultilayerPerceptronClassificationModel(uid, mlpModel.weights)
+    model.setLayers(myLayers)
```

Comment:
hmm, do we really have exposed parameters into the model? The purpose of this PR is to have training parameters in the model, but except for layers, seems other parameters are not set?

Reply:
The params are set at line 152 in Preditor

```copyValues(train(casted).setParent(this))``` 

It copies all the params in ```paramMap``` from ```Preditor``` (```MultilayerPerceptronClassifier```) to the model. Actually since ```layers``` is one of the params now, it also sets here. No need to set it explicitly. I will remove line 251.